packages/node_modules/@webex/internal-plugin-conversation/src/conversation.js

```diff
@@ -442,7 +442,7 @@ const Conversation = SparkPlugin.extend({
           objectType: 'activity',
           clientTempId: uuid.v4(),
           actor: this.spark.internal.device.userId
-        }, pick(activity, 'activityType', 'parentActivityId'));
+        }, pick(activity, 'activityType', 'parentActivityId', 'parent'));
```

Comment:
@willzxd I actually just updated the test for this. Did you verify it works with this change?

Reply:
I saw that. 

Current server behavior is client side is able to post thread replies that contain `activityType` and `parentAcitivtyId` and/or `parent`. 
1 - 2 month later server behavior will change. It will only accept post thread replies that contain `parent`.
The server side will always return responses that contain `parent`.

So we need to be compatible with both to avoid breaking changes.
Updated wiki here: https://wiki.cisco.com/pages/viewpage.action?pageId=236794024